modules/nf-core/merquryfk/merquryfk/meta.yml

```
@@ -50,54 +58,30 @@ output:
       type: file
       description: Assembly only kmer positions not supported by reads in bed format
       pattern: "*_only.bed"
-  - spectra_cn_fl_png:
-      type: file
-      description: "Unstacked copy number spectra filled plot in PNG format"
-      pattern: "*.spectra-cn.fl.png"
-  - spectra_cn_ln_png:
-      type: file
-      description: "Unstacked copy number spectra line plot in PNG format"
-      pattern: "*.spectra-cn.ln.png"
-  - spectra_cn_st_png:
-      type: file
-      description: "Stacked copy number spectra line plot in PNG format"
-      pattern: "*.spectra-cn.st.png"
-  - spectra_asm_fl_png:
-      type: file
-      description: "Unstacked assembly spectra filled plot in PNG format"
-      pattern: "*.spectra-asm.fl.png"
-  - spectra_asm_ln_png:
+  - spectra_cn_fl:
       type: file
-      description: "Unstacked assembly spectra line plot in PNG format"
-      pattern: "*.spectra-asm.ln.png"
-  - spectra_asm_st_png:
+      description: "Unstacked copy number spectra filled plot in PNG or PDF format"
+      pattern: "*.spectra-cn.fl.*"
```

```suggestion
      pattern: "*.spectra-cn.fl.{png,pdf}"
```
This should also be applied to the other patterns too.